### ISSUE TITLE
issue with installation on Mac OS

### ISSUE THREAD

**Issue by 3DMiller** [introlab/rtabmap#537]
When downloading the dependencies via Macports through the command line, the error appears:  "Error: Failed to configure libpcl: vtk must be installed with +qt5" . Any ideas on what should be done? I am a beginner user, so not a lot of experience running things in the Terminal.

**Comment by matlabbe** [comment]
You may try:
```
sudo port install vtk +qt5
```

**Comment by 3DMiller** [comment]
Hi, 

I think that worked. I’m sorry for the question being very elementary, but if I already downloaded RTAB Map from the dmg file, do I need to run the commands in the terminal? Or, will RTAB Map know where to locate the dependencies to use them? I think qt% worked and I downloaded the rest through Macports. Is the installation complete?


Thanks,

Spencer Miller
Masters student in Marine Science - Hawaiʻi Pacific University 
www.millerspencer.weebly.com <http://www.millerspencer.weebly.com/>
www.sketchfab.com/S_Miller <http://www.sketchfab.com/S_Miller> 
https://jmadinlab.github.io/people/spencer <https://jmadinlab.github.io/people/spencer>
